.gitignore

```diff
@@ -30,3 +30,4 @@ classes
 
 # Used to check reproducibility of Mockito jars
 checksums/
+local.properties
```

Comment:
This does not seem necessary, as Mockito doesn't create this file, nor should our test suite. Was this addition intentional? If not, please remove it.

Reply:
Will do.